### PR TITLE
Modifications to .gitignore and config/admins.txt

### PR DESCRIPTION
**Pull request by coiax**
Modified .gitignore, and added myself to the admins file so git will stop whining to me about the dirty working environment.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,3 +10,173 @@
...
+
+### https://raw.github.com/github/gitignore/cc542de017c606138a87ee4880e5f06b3a306def/Python.gitignore
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
```
Comment (MrStonedOne) [inline .gitignore]: Nope, these can generally be needed to be distributed.

File: .gitignore [hunk 2]
```diff
@@ -10,3 +10,173 @@
...
+Desktop.ini
+
+# Recycle Bin used on file shares
+$RECYCLE.BIN/
+
+# Windows Installer files
+*.cab
+*.msi
+*.msm
+*.msp
```
Comment (MrStonedOne) [inline .gitignore]: and if we need to package an installer?

Comment (coiax) [inline .gitignore]: 1) Why are we distributing binaries via our git repo, which we'll never be able to remove, and will only make the repo bigger as they change?
2) You can always force add something to git even if it's in the .gitignore file.

File: .gitignore (hunk 1 above)
Comment (coiax) [inline .gitignore]: Literally running the map merger files earlier left me with a __pycache__ directory, which is where python3 leaves all of its compiled stuff now.

Comment (MrStonedOne) [inline .gitignore]: I mean the *.so

File: .gitignore (hunk 2 above)
Comment (MrStonedOne) [inline .gitignore]: You haven't taken a look at /tools/ have you?

Comment (coiax) [inline .gitignore]: HUSH IF I DON'T ACKNOWLEDGE THEM THEY DON'T EXIST

File: .gitignore (hunk 1 above)
Comment (coiax) [inline .gitignore]: Could remove the *.so if they're only going to occur in the repo when we want them to be there.

**Pull request merged**